ykman-gui/qml/ContentStack.qml

```diff
@@ -50,44 +51,70 @@ StackView {
         return false
     }
 
+    function popToDepth(height) {
+        pop(find(function(item, searchIndex) {
+            return searchIndex === height;
+        }))
+    }
+
+    function replaceAtDepth(depth, item, objectName) {
```

Comment:
This signature is a bit unclear to me, what does the objectName parameter do? We also currently never call the function it without it.

Reply:
Would `newObjectName` be clearer?

It's used to detect if the current item at the specified depth is the item we want to replace to, and in that case we only pop down to that depth instead of replacing anything.

It seems to me like the `objectName` property isn't available on the `item` argument - I think because that `item` is the wrapping `Component`, not the `*View` which has the `objectName` property - so that's why we need to pass it in. I would like to get rid of it but didn't see a good way to do so.

Reply:
Renamed to `newObjectName`.